### ISSUE TITLE
Icon Accessibility

### ISSUE THREAD

**Issue by JyjyStudio** [TeamWertarbyte/material-ui-search-bar#138]
Hello,
is it possible to add aria-labels, role="presentation" and focusable="false" to these icons for improve accessibility ?

When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

![accessibility](https://user-images.githubusercontent.com/75069243/211556147-3536a61c-6cf7-4fef-8e98-95578d056e40.png)
